client/src/contexts/AppContext.ts

```diff
@@ -21,12 +27,15 @@ interface AppContextInterface {
   voteCounts: OptionalVoteCounts
   usbDriveStatus: string
   usbDriveEject: () => void
+  addPrintedBallot: (arg0: PrintedBallot) => void
```

Comment:
nit: Easier to search for when same param names are used…
```suggestion
  addPrintedBallot: (printedBallot: PrintedBallot) => void
```